src/Socket_p.h

```diff
@@ -573,7 +573,7 @@ namespace Arcus
 
         {
             std::lock_guard<std::mutex> lk(receiveQueueMutexBlock);
```

Comment:
This one is superfluous as well, I think (see last comment about almost the same situation earlier in this same file).